perfkitbenchmarker/pkb.py

```diff
@@ -315,13 +305,17 @@ def RunBenchmarks(publish=True):
 
   try:
     benchmark_list = benchmark_sets.GetBenchmarksFromFlags()
+    total_benchmarks = len(benchmark_list)
+    sequence_range = range(total_benchmarks, 0, -1)
```

Comment:
Yes. RunThreaded uses list.pop(), which dequeues tasks from the back of the list. Therefore, the last item in the list needs to be labeled as task 1, and the first item in the list needs to be labeled as task N.